ops/docker/op-stack-go/Dockerfile

```diff
@@ -9,6 +9,12 @@
 # It will default to the target platform.
 ARG TARGET_BASE_IMAGE=alpine:3.20
 
+# The ubuntu target base image is used for the op-challenger build with kona and asterisc.
+ARG UBUNTU_TARGET_BASE_IMAGE=ubuntu:20.04
+
+# The version of kona to use.
+ARG KONA_VERSION
```

Comment:
Maybe we just supply a dummy default value here so it doesn't always have to be set? e..g `ARG KONA_VERSION=none`

Reply:
Ah good idea!